README.md

```diff
@@ -346,6 +346,7 @@ Deaf and Hard of Hearing accessibility resources:
 - [Dogsheep](https://dogsheep.github.io) [<img src="https://raw.githubusercontent.com/humanetech-community/awesome-humane-tech/main/logo/github.svg?sanitize=true" width="16"/>](https://github.com/dogsheep) - Export data from big companies (e.g. Google, Twitter) into Sqlite for personal analytics.
 - [Tracking Exposed](https://tracking.exposed/) [<img src="https://raw.githubusercontent.com/humanetech-community/awesome-humane-tech/main/logo/github.svg?sanitize=true" width="16"/>](https://github.com/tracking-exposed) - Research, browser addons on tracking, profiling, and influence of algorithms.
 - [Social Media Awareness Toolkit](https://www.smat-app.com/about) [<img src="https://raw.githubusercontent.com/humanetech-community/awesome-humane-tech/main/logo/gitlab.svg?sanitize=true" width="16"/>](https://gitlab.com/smat-project) - An open and intuitive way to scrutinize what’s trending on internet platforms.
+- [Deletion Day](https://deletionday.com) [<img src="https://raw.githubusercontent.com/humanetech-community/awesome-humane-tech/main/logo/github.svg?sanitize=true" width="16"/>](https://github.com/deletionday/site) – A holiday to raise awareness of the myriad privacy concerns of today’s Internet. Deletion Day takes place on April 4th every year.
```

Comment:
Hi @deletionday .. though you do not _really_ provide an open source software project here (other than the website code), I really like the idea this is based on.

There's one requirement however, and that is that each list entry fits on a single line (at larger resolutions). Could you change your description to _"A holiday on April 4th to raise awareness of the myriad privacy concerns of today’s Internet."_? That would fit.

Also a normal dash `-` must be used, not the longer version (forgot the char name).

Reply:
PS. I don't know if this is your intention, but on Firefox most of the site title 'Deletion Day' is cut off i.e. above the site top.

Reply:
Thanks for the review, @aschrijver! I just updated the description re: your feedback.

> PS. I don't know if this is your intention, but on Firefox most of the site title 'Deletion Day' is cut off i.e. above the site top.

The site title should have a single line cut off, followed by a visible line that’s being deleted:

<img width="447" alt="image" src="https://user-images.githubusercontent.com/61673853/112886007-3c807100-9086-11eb-941c-1f7d9ed90c2f.png">

(Unless your OS “Reduce Motion” setting is turned on, in which case nothing should be cut off)

Thanks again!

Reply:
Ah, then it is alright. This is the way it looked in my browser. Thanks for making the change. I will merge the PR.